app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,30 @@
+class RestaurantsController < ApplicationController
+
+  include Response
+  include ActiveModel
+  def index
+    @restaurants = Restaurant.all
+    # Cuisine.find(@restaurants.cuisine_id)
+    # render :json => @restaurants.to_json(:except => [:created_at, :updated_at], include: 'cuisine' )
+    render :json => @restaurants.to_json(:except => [:created_at, :updated_at, :cuisine_id], :include => {:reviews => {:except => [:created_at, :updated_at, :restaurant_id]}, :cuisine => {:except => [:created_at, :updated_at]}})
+  end
+
+  def averageRating
```

Comment:
please use snake_case for methods
in general, you can look at our style guide for reference - https://github.com/WeConnect/style-guide

Reply:
you can also install rubocop to give you annoying comments about style